client/src/components/Workbench/index.js

```diff
@@ -1,18 +1,17 @@
 import { hot } from 'react-hot-loader/root';
 import React from 'react';
-import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 import Workbench from './Workbench';
 import ManageAccount from '../ManageAccount';
 import * as ROUTES from '../../constants/routes';
-import TicketUnauthenticatedCreate from '../Tickets/TicketUnauthenticatedCreate';
+import TicketStandaloneCreate from '../Tickets/TicketStandaloneCreate';
 
 function AppRouter() {
   return (
     <Router>
-      <Route path="/accounts/profile" component={ManageAccount} />
       <Route path={ROUTES.WORKBENCH} component={Workbench} />
-      <Route path="/tickets/new" component={TicketUnauthenticatedCreate} />
-      <Redirect from="/accounts" to="/accounts/profile" />
```

Comment:
Any reason to remove this redirect?

Reply:
Removed redirect as causing problems. I'll address in FP-734 in the sense that the route will change.